app/controllers/miq_ae_customization_controller/dialogs.rb

```diff
@@ -1315,6 +1329,11 @@ def dialog_set_record_vars(dialog)
                     fld[:default_value] = field[:default_value]
                     fld[:required] = field[:required]
 
+                  elsif ["DialogFieldDropDownList"].include?(field[:typ])
+                    fld.update(
+                      :force_multi_value => field[:multi_value],
+                    )
```

Comment:
I'm not super familiar with this file, is there a reason this one is a `fld.update` instead of just `fld[:force_multi_value] =`?